### PR TITLE
Update scala 2.12 and 2.13

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

**Comment by regadas** [comment]
build fails due to `scoverage` https://github.com/scoverage/sbt-scoverage/issues/319

**Comment by johnynek** [comment]
Huh, the micro site is failing somehow. I haven't checked the logs.

**Comment by regadas** [comment]
Right, the error is related to `tut` which is removed in #920

**Comment by johnynek** [comment]
so, if we merge #920 this should be good to go?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
           - 11
         scala:
           - 2.11.12
-          - 2.12.12
+          - 2.12.13
```
Comment (johnynek) [inline .github/workflows/ci.yml:63]: shouldn't we add 2.13 to this list?

Comment (regadas) [reply .github/workflows/ci.yml:63]: @johnynek sorry for the late reply, hard couple of days. Indeed! I guess we forgot to add it when we release 2.13.x.

**Review by johnynek — APPROVED** [review]
This is great.

**Pull request merged**